templates/repo/projects/view.tmpl

```diff
@@ -100,6 +100,12 @@
 												<label for="new_board_title">{{$.i18n.Tr "repo.projects.board.edit_title"}}</label>
 												<input class="project-board-title" id="new_board_title" name="title" value="{{.Title}}" required>
 											</div>
+											<div class="field">
+												<label for="set_board_default">Set this board as Default one</label>
```

Comment:
Need to localize this.

Reply:
@silverwind the whole field si not recognized somehow :/
backend: works
frontend: early stages

do you have an idear why https://github.com/go-gitea/gitea/pull/14147/files#diff-b19908ffc888ff94458704d923f4e1f3bd07e43d385b3a8ae9eb080021223e30R39 does not contain the input value?

Reply:
never mind, I'll change the UI by adding a new entry & endpoint